python-sdk/tests_integration/sql/test_table.py

```diff
@@ -79,6 +79,20 @@
             f"file://{socket.gethostbyname(socket.gethostname())}:22",
             f"file://{socket.gethostbyname(socket.gethostname())}:22/tmp/duckdb.db.test_tb",
         ),
+        (
+            Connection(
+                conn_id="test_mssql",
+                conn_type="mssql",
+                host="astroserver.database.windows.net",
```

Comment:
Do we need to use actual servername here (or) a fake name would suffice?

Reply:
Fake name should be fine for this test.

Reply:
so blur the server name in the attached screenshot from Marquez as well please

Reply:
Done.